### DISCUSSION TITLE
Gallery extensions shows error: Can't write file

### DISCUSSION THREAD

**Discussion by sorenpeter** [datenstrom/community#546] [Report a bug] [answered]
Hi,

I'm trying to use the gallery, but I'm getting an error when I try to use thumbnails:

This markdown: `[gallery foeg2020_MGC/ - 150]` gives this error:

```
Server error:

Something went wrong. Can't write file 'media/thumbnails/foeg2020_MGC-20200912_214649_49-150x150.jpg'!
```

If it just use `[gallery foeg2020_MGC/]` then I get the images one by one, but no titled thumbnails.

I have tried to change the permissions for the folders on the server to 755, but it does not seem to help.
I don't have SSH access to my server, so I use FileZilla or connect via WebDAV in MacOS Finder.

Any ideas how to fix this issue?

/Søren Peter

**Comment by sorenpeter** [comment]
I found the issue: i had a jpg with size zero, due to bad upload.

**Comment by markseuffert** [comment, marked as answer]
I could reproduce the problem and added an aditional check to the image extension. Thank you.
